src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
+---
+```
+
+## Introduction section
+
+After you insert the front matter, insert the introduction, which typically just includes text and an image. If you need help with inserting images, see the example at the bottom.
+
+## Tiles
+
+Tiles are a series of boxes that contain the main subject areas for your product. You should just list these in order you want them to appear, and the cascading style sheet will render them across the page.
+
+Here's an example of a tile: 
+
+```
+<LandingPageTileGrid>
+  <LandingPageTile
+    title="INSERT_YOUR_BOX_TITLE_HERE."
+    icon="fe-INSERT_THE_ICON_NAME"
+  >
+
+Insert tile content here...
+
+</LandingPageTile>
+```
+For each tile, do the following:
+
+* Insert a value for `title` that explains the purpose of the category.
+* Insert a value for `icon` by prefixing one of the [Feather Icons](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) with `fe-` so it is recognized (for example, fe-alert-triangle).
+* Between the `LandingPageTile` tags, insert text, such as a bullet list with links to product documentation.
+
+## Button for viewing all docs in the category
+
+After your tiles, you should have a single button offers to take users to all the documentation for that category. 
+
+Here's an example:
+
+```
+<ButtonLink
+  role="button"
+  to="INSERT_LINK_TO_DIRECTORY_WITH_ALL_THESE_DOCS"
```

Comment:
It might be best to mention that the "table of contents" page that gets linked here will always be at the same path as the landing page, but with `/table-of-contents` appended to it. These table of contents pages get built automatically for every landing page. For example, if this landing page was located at `/docs/apm`, this link should be `/docs/apm/table-of-contents`.

Reply:
OK, I essentially inserted your comment into that section to explain how the link works.